src/stan/variational/advi.hpp

```diff
@@ -334,6 +520,14 @@ namespace stan {
 
         // draw more samples from posterior and write on subsequent lines
         if (out_stream_) {
```

Comment:
This is all going to get refactored, so don't sweat it too much at this point.
Usually when we have two streams, one is for output to a CSV file and one is
printing from the model and exceptions like warnings that pop up.


> On Aug 26, 2015, at 12:35 AM, Daniel Lee <notifications@github.com> wrote:
> 
> In src/stan/variational/advi.hpp:
> 
> > @@ -334,6 +520,14 @@ namespace stan {
> >  
> >          // draw more samples from posterior and write on subsequent lines
> >          if (out_stream_) {
> 
> I feel like I'm missing something: why is there an out_stream_ and print_stream_ here for this if statement? I could understand one or the other.
> 
> —
> Reply to this email directly or view it on GitHub.
>